Makefile

```diff
@@ -1,14 +1,26 @@
 DEVVM=work
 DEVDIR=/home/user/projects/securedrop-workstation # important: no trailing slash
+HOST=$(shell hostname)
 
-all: validate clean sd-whonix sd-svs sd-gpg sd-journalist sd-decrypt \
+assert-dom0:
+ifneq ($(HOST),dom0)
+	@echo "     ------ securedrop-workstation's makefile must be used only on dom0! ------"
+	exit 1
+endif
```

Comment:
Since the new "remove sd-journalist" step involves an `rm -rf ...`, I wanted to put at least a bit of logic to try to avoid removing the files in the wrong place or on the wrong VM. Thus this step, and using it in various recipes.